src/Pact/Typechecker.hs

```diff
@@ -875,8 +905,10 @@ toAST (TApp Term.App{..} _) = do
   case fun of
 
     FDefun {..} -> do
+      if null _fBody
+      then assocNode _fRetId n -- TODO: is this necessary?
```

Comment:
Would be nice to know if it is indeed necessary, as otherwise we have  `_fRetId` lying around if it's not. But fine to leave it in.

Reply:
Let's punt on this for now, as we want to release today